styles/sections/_nav.scss

```diff
@@ -2,32 +2,37 @@
 
 // Top Navigation Bar
 #top-nav {
-  background: colors.$color-blue;
-  padding: 16px 0;
+  background-color: colors.$color-blue;
   position: sticky;
   top: 0;
   left: 0;
   z-index: 99;
   width: 100%;
-
-  @container body (min-width: 1024px) {
-    padding: 20px 0;
-  }
 }
 
 .container {
   display: flex;
   justify-content: flex-start;
   align-items: center;
-  padding: 0 24px;
+  padding: 16px 24px;
+  background: radial-gradient(
+    circle at -100px -110px,
+    colors.$color-teal-50,
+    colors.$color-blue 500px
+  );
 
   @container body (min-width: 1024px) {
-    padding: 0 150px;
+    padding: 20px 150px;
     justify-content: space-between;
+    background: radial-gradient(
```

Comment:
I think we can reduce this background down to `background-position: -100px -200px;` since the position is the only thing that's changing (it worked in Chrome at least.)

Reply:
I like your idea of removing repetitive code, I'm not sure I follow though? The size also changes from 500px to 750px

Reply:
Oh dang, I missed that change. Nevermind, ignore me lol